contracts/ZKPVerifier.sol

```diff
@@ -51,40 +54,60 @@ contract ZKPVerifier is IZKPVerifier, Ownable {
         return true;
     }
 
-    function getZKPRequest(uint64 requestId)
-        external
-        view
-        override
-        returns (ICircuitValidator.CircuitQuery memory)
-    {
+    function getZKPRequest(
+        uint64 requestId
+    ) external view override returns (ICircuitValidator.CircuitQuery memory) {
```

Comment:
Why not to return the validator address as well to get more info about the request id? It there are not release dependencies, it should not cost us much.